scripts/backfill-project-areas.sh

```
@@ -0,0 +1,395 @@
+
+if [[ "$APPLY" == "false" && $ASSIGNED -gt 0 ]]; then
+  echo ""
+  echo "This was a dry run. Re-run with --apply to make changes."
+fi
+
+# ── Strip area:* labels from issues ──────────────────────────────────────────
+if [[ "$STRIP_LABELS" == "true" ]]; then
+  echo ""
+  echo "Scanning for area:* labels to remove from issues..."
+  STRIPPED=0
+  STRIP_SKIPPED=0
```

Shellcheck (0.11.0)

[warning] STRIP_SKIPPED appears unused. Verify use (or export if used externally).

(SC2034)